maven/bnd-testing-maven-plugin/src/main/java/aQute/bnd/maven/testing/plugin/TestingMojo.java

```diff
@@ -44,7 +34,7 @@
 	@Parameter(readonly = true, required = true)
 	private List<File>	bndruns;
 
-	@Parameter(defaultValue = "${project.build.directory}", readonly = true)
+	@Parameter(defaultValue = "${project.build.directory}/tmp", readonly = true)
```

Comment:
As I discussed above, we need 2 args: cwd to run tests in and reportsDir. I am sure `target/tmp` is a bad choice. We need a basedir to use for cwd of each bndrun file and the use the bndrun file name for a subfolder.

"${project.build.directory}/test/${bndrun}"  for cwd and "${project.build.directory}/test-reports/${bndrun}" for reports dir. So the parameters to this plugin are "${project.build.directory}/test" and "${project.build.directory}/test-reports"